src/main/java/org/broadinstitute/hellbender/tools/walkers/SplitIntervals.java

```diff
@@ -147,8 +147,9 @@ public void onTraversalStart() {
                         })  // turn the intervals back into an IntervalList
                 ).collect(Collectors.toList());
 
-        final DecimalFormat formatter = new DecimalFormat("0000");
-        IntStream.range(0, scatteredFinal.size()).forEach(n -> scatteredFinal.get(n).write(new File(outputDir, formatter.format(n) + extension)));
+        final int maxNumberOfPlaces = Math.max((int)Math.floor(Math.log10(scatterCount-1))+1,1); //max because single scatter case bombs otherwise
```

Comment:
Should we keep the default at 4 to save anyone who has stupid scripts that look for 0001?

Reply:
I suppose that's a fair point.  Will do.